.eslintrc.js

```diff
@@ -17,7 +17,9 @@ module.exports = {
         'react/no-access-state-in-setstate': 'off', // fixme
         'react/no-array-index-key': 'off', // fixme
         'react/no-this-in-sfc': 'off',
+        'formatjs/enforce-description': 'error' // fixme
     },
+    plugins: ['formatjs'],
```

Comment:
Is this something new that we're doing in addition to prettier? Should it be a separate PR from the dependencies upgrade or is it needed due to changes in babel and/or react-intl?

Reply:
I removed it, doesn't work well for us right now

Reply:
@jstoffan I added this back, they fixed that above issue. There is 1 remaining issue I filed https://github.com/formatjs/formatjs/issues/304 which only effects 1 of our files.

In any case, I will do a followup PR to remove this plugin from here and move to frontend repo.
BTW, yeah its needed with the new babel react intl plugin which is also now upgraded